api_app/models/domain/airlock_resource.py

```diff
@@ -0,0 +1,36 @@
+from enum import Enum
+from typing import List
+from pydantic import Field
+from models.domain.azuretremodel import AzureTREModel
+from resources import strings
+
+
+class AirlockResourceType(str, Enum):
+    """
+    Type of resource to deploy
+    """
+    AirlockRequest = strings.AIRLOCK_RESOURCE_TYPE_REQUEST
+    # TBD Airlock review
+
+
+class AirlockResourceHistoryItem(AzureTREModel):
```

Comment:
are we going to implement history?

Reply:
If we follow the convention, in the "base" resource, history is implemented there